README.md

```diff
@@ -176,6 +176,22 @@ PINRemoteImageManager.shared().setAuthenticationChallenge { (task, challenge, co
 }
 ```
 
+### Support for high resolution images
+Currently there are two ways PINRemoteImage is supporting high resolution images:
+1. If the URL will contains a `@2x` or a `@3x` postfix it will automatically handled from PINRemoteImage and the resulting image will be returned with the right scale.
```

Comment:
nit: If the URL contains an `@2x` or an `@3x` postfix it will be automatically handled by PINRemoteImage and the resulting image will be returned at the right scale.